src/bcd.c

```diff
@@ -909,12 +910,12 @@ bcd_backtrace_process(struct bcd_session *session)
 		const char *cargv[BCD_ARGC_LIMIT];
 	} u;
 	bcd_error_t error;
-	size_t delta = 2;
+	size_t delta = 0;
 	ssize_t r;
 
-	u.cargv[0] = strdup(bcd_config.invoke.path);
+	u.cargv[delta++] = strdup(bcd_config.invoke.path);
 
-	r = bcd_arg_get(&(u.argv[delta]), sizeof(u.argv) / sizeof(*u.argv) - delta,
+	r = bcd_arg_get(&(u.argv[delta]), sizeof(u.argv) / sizeof(*u.argv) - 2,
```

Comment:
We need to account for the NULL terminator as well here like is done when adding KVs.